carmin.yaml

```diff
@@ -146,6 +164,54 @@ definitions:
         type: string
       supportedAPIVersion:
         type: string
+  
+  
+  
+  PipelineParameter:
+    type: object
+    required:
+      - name
+      - paramType
+      - isOptional
+      - isReturnedValue
+    properties:
+      name:
+        type: string
+      paramType:
+        type: string
+      isOptional:
+        type: boolean
+      isReturnedValue:
+        type: boolean
+      defaultValue:
+        type: string
```

Comment:
I think it can be any type here. Therefore I simply remove the definition `type` here. It can pass the swagger check. By the way, just let you know this pull-request has been closed since I will merge this branch with my branch of execution.